apps/vs-code-designer/src/assets/UnitTestTemplates/TestClassFile

```diff
@@ -1,77 +1,128 @@
-        private UnitTestEngine UnitTestExecutor;
+        /// <summary>
+        /// The root path of the workspace project directory.
+        /// </summary>
+        private readonly string rootPath;
 
-        public <%= unitTestName %>()
-        {
-            this.UnitTestExecutor = new UnitTestEngine();
-            this.WorkflowDefinition = File.ReadAllText(@"./<%= workflowName %>/workflow.json");
-            var mockData = JObject.Parse(File.ReadAllText(@"<%= workflowName %>Mocks.json"));
-            this.Mocks = mockData.ToObject<Dictionary<string, OperationMock>>();
-        }
+        /// <summary>
+        /// The full path to the workflow JSON file.
+        /// </summary>
+        private readonly string workflowDefinitionPath;
 
-        [TestMethod]
-        public void ExecuteWorkflow_WithMockTrigger_ShouldSucceed()
+        /// <summary>
+        /// The full path to the connections JSON file.
+        /// </summary>
+        private readonly string connectionsPath;
+
+        /// <summary>
+        /// The full path to the parameters JSON file.
+        /// </summary>
+        private readonly string parametersPath;
+
+        /// <summary>
+        /// The full path to the local settings JSON file.
+        /// </summary>
+        private readonly string localSettingsPath;
+
+        /// <summary>
+        /// The mocked trigger outputs that are generated from a workflow run.
+        /// </summary>
+        public TriggerMock triggerMock;
+
+        /// <summary>
+        /// The mocked actions' outputs that are generated from a workflow run.
+        /// </summary>
+        public Dictionary<string, ActionMock> actionMocks;
+
+        /// <summary>
+        /// Initializes a new instance of the class named <see cref="<%= UnitTestName %>" />.
+        /// This constructor loads the necessary paths for the workflow, connections, parameters, and local settings JSON files.
+        /// The constructor also loads the mock data from the file named <%= UnitTestName %>_mock.json.
+        /// </summary>
+        public <%= UnitTestName %>()
         {
-            // Arrange – define mocks either from a static source or as a lambda function
-            var triggerMock = new OperationMock("When_a_HTTP_request_is_received", FlowStatus.Succeeded, new JObject()
-            {
-                { "body", new JObject() { { "isReleased", true }, { "isEnabled", false }, { "isFlag", false } } }
-            });
-
-            // Act 
-            var unitTestFlowRun = this.UnitTestExecutor.ExecuteWorkflow(this.WorkflowDefinition, this.Mocks, triggerMock).Result;
-
-            // Assert
-            Assert.IsNotNull(unitTestFlowRun);
-            Assert.AreEqual(FlowStatus.Succeeded, unitTestFlowRun.Properties.Status);
-            Assert.AreEqual("myvalue", unitTestFlowRun.Properties.Actions["Get_Rows_Receipt"].Outputs.Value<string>());
+            //Get the root path of the workspace
```

Comment:
nit: space after start of comment.

and "Set the root path of the workspace.

Reply:
fixed